simulation_ws/.rosinstall

```diff
@@ -1,3 +1,2 @@
-# kinetic.rosinstall
-- setup-file: {local-name: /opt/ros/kinetic/setup.sh}
-- git: {local-name: src/turtlebot3_simulations, uri: "https://github.com/ROBOTIS-GIT/turtlebot3_simulations", version: 1.2.0}
\ No newline at end of file
+- git: {local-name: src/aws-robomaker-bookstore-world, uri: "https://github.com/aws-robotics/aws-robomaker-bookstore-world.git", version: v0.0.1}
+- git: {local-name: src/turtlebot3_simulations, uri: "https://github.com/ROBOTIS-GIT/turtlebot3_simulations", version: 1.2.0}
```

Comment:
while we're making this change could we also move this turtlebot3_simulations to `src/deps/` instead to be consistent with where robot_ws puts its dependencies (https://github.com/aws-robotics/aws-robomaker-sample-application-voiceinteraction/blob/master/robot_ws/.rosinstall)